spec/models/avalon_marker_spec.rb

```diff
@@ -0,0 +1,64 @@
+# Copyright 2011-2015, The Trustees of Indiana University and Northwestern
+#   University.  Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed
+#   under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+#   CONDITIONS OF ANY KIND, either express or implied. See the License for the
+#   specific language governing permissions and limitations under the License.
+# ---  END LICENSE_HEADER BLOCK  ---
+
+require 'spec_helper'
+
+describe AvalonMarker, type: :model do
+  describe 'abilities' do
+    subject { ability }
+    let(:ability) { Ability.new(user) }
+    let(:user) { FactoryGirl.create(:user) }
+    let(:master_file) { FactoryGirl.create(:master_file) }
+    let(:avalon_marker) { FactoryGirl.create(:avalon_marker, playlist_item: playlist_item) }
```

Comment:
This refers to playlist_item, which is defined in the next line. The two lines should be flipped, I think.